### PR TITLE
[RW-4971][RISK=NO]Fix grammatical mistakes - NIH findings

### PR DESCRIPTION
**Pull request by NehaBroad**
NIH Found some grammatical mistakes:  
Have created just one PR for all fix so its easier for cherry picking in case it does not make it to weekly release 
This PR contains fix for all the points the screen shot for some are as follows.

- Fix for 7, 8 , 9 ,10 ,11, 12

Screen Shot 2020-05-18 at 4 25 59 PM

Screen Shot 2020-05-18 at 4 26 44 PM
Screen Shot 2020-05-18 at 4 27 31 PM
Screen Shot 2020-05-18 at 4 29 50 PM

-  Issue Number: 3, 6, 19, 20, 21, 22, 23

Screen Shot 2020-05-18 at 6 46 24 PM

-  Fix  for Issue: 24, 25, 26, 27, 28

Screen Shot 2020-05-18 at 4 38 00 PM
Screen Shot 2020-05-18 at 4 39 25 PM

- Fix for Issue 29, 30 ,31, 32 ,33 
Screen Shot 2020-05-18 at 4 40 19 PM
Screen Shot 2020-05-18 at 4 40 30 PM
Screen Shot 2020-05-18 at 4 40 42 PM

- Issue number: 34, 35 , 36 , 37 and 38

Screen Shot 2020-05-18 at 6 56 11 PM

Screen Shot 2020-05-18 at 4 22 34 PM

- Fix for 39,40

Screen Shot 2020-05-18 at 4 23 09 PM

-  Fix for 42, 43,44 and 445

Screen Shot 2020-05-18 at 6 47 47 PM

 - Fix for 49
Screen Shot 2020-05-18 at 4 26 19 PM

---
**PR checklist**

- [x] This PR meets the Acceptance Criteria in the JIRA story
- [x] The JIRA story has been moved to Dev Review
- [x] This PR includes appropriate unit tests
- [x] I have run and tested this change locally
- [ ] I have run the E2E tests on ths change against my local UI + API server with `yarn test-local`
- [ ] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer
- [ ] If this includes an API change, I have updated the appropriate Swagger definitions and notified API consumers
- [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later

### CODE REVIEW

File: ui/src/app/pages/homepage/homepage.tsx [hunk 1]
```diff
@@ -355,8 +355,8 @@ export const Homepage = withUserProfile()(class extends React.Component<Props, S
                                     <CustomBulletList>
                                       <CustomBulletListItem bullet='→'>
                                         Create a <StyledAnchorTag href='https://support.google.com/chrome/answer/2364824'
-                                          target='_blank'>Chrome Profile</StyledAnchorTag> with your <i>All of Us</i> Researcher
-                                        Workbench Google account. This will keep your Workbench browser sessions isolated from
+                                          target='_blank'>Chrome Profile</StyledAnchorTag> with your <AoU/> Researcher
+                                        Workbench Google account. This will keep your workbench browser sessions isolated from
```
Comment (jmthibault79) [inline ui/src/app/pages/homepage/homepage.tsx:359]: is this issue 9?

Comment (NehaBroad) [reply ui/src/app/pages/homepage/homepage.tsx:359]: Issue number 18 

RW Welcome page | Grammar&Punctuation | Under tips to get your started, Workbench should be lowercase or referred to as "Researcher Workbench" (2nd sentence in first bullet)
-- | -- | --

File: ui/src/app/pages/data/data-page.tsx [hunk 2]
```diff
@@ -70,7 +70,7 @@ enum Tabs {
 }
 
 const descriptions = {
-  datasets: `A dataset is a table containing data about a Cohort that can
+  datasets: `A dataset is a table containing data about a cohort that can
```
Comment (NehaBroad) [inline ui/src/app/pages/data/data-page.tsx:73]: Row# 45

File: ui/src/app/pages/homepage/homepage.tsx [hunk 3]
```diff
@@ -333,7 +333,7 @@ export const Homepage = withUserProfile()(class extends React.Component<Props, S
                                 style={{alignSelf: 'flex-end', color: colors.accent, cursor: 'pointer'}}
                                 onClick={() => navigate(['workspaces'])}
                               >
-                                See all Workspaces
+                                See all workspaces
```
Comment (NehaBroad) [inline ui/src/app/pages/homepage/homepage.tsx:336]: Row # 46

File: ui/src/app/pages/homepage/recent-workspaces.tsx [hunk 4]
```diff
@@ -46,7 +46,7 @@ export const RecentWorkspaces = withUserProfile()
       {this.state.loading && <SpinnerOverlay dark={true} />}
       {this.state.recentWorkspaces.length === 0 && !this.state.loading ?
         <div style={{color: colors.primary, margin: '.5em 2em'}}>
-          <h2 style={{fontWeight: 600, lineHeight: 1.5}}>Create your first Workspace</h2>
+          <h2 style={{fontWeight: 600, lineHeight: 1.5}}>Create your first workspace</h2>
```
Comment (NehaBroad) [inline ui/src/app/pages/homepage/recent-workspaces.tsx:49]: # 17

File: ui/src/app/pages/homepage/registration-dashboard.tsx [hunk 5]
```diff
@@ -140,7 +141,7 @@ export const getRegistrationTasks = () => serverConfigStore.getValue() ? ([
     completionPropsKey: 'eraCommonsLinked',
     loadingPropsKey: 'eraCommonsLoading',
     title: 'Connect Your eRA Commons Account',
-    description: 'Connect your Workbench account to your eRA Commons account. There is no exchange of personal data in this step.',
+    description: 'Connect your Researcher Workbench account to your eRA Commons account. There is no exchange of personal data in this step.',
```
Comment (NehaBroad) [inline ui/src/app/pages/homepage/registration-dashboard.tsx:144]: # 14

File: ui/src/app/pages/homepage/registration-dashboard.tsx [hunk 6]
```diff
@@ -152,7 +153,7 @@ export const getRegistrationTasks = () => serverConfigStore.getValue() ? ([
     completionPropsKey: 'trainingCompleted',
     title: <span><i>All of Us</i> Responsible Conduct of Research Training</span>,
     description: <div>Complete ethics training courses to understand the privacy safeguards and the
-      compliance requirements for using the <i>All of Us</i> Dataset.</div>,
+      compliance requirements for using the <AoU/> dataset.</div>,
```
Comment (NehaBroad) [inline ui/src/app/pages/homepage/registration-dashboard.tsx:156]: # 15

File: ui/src/app/pages/homepage/registration-dashboard.tsx [hunk 7]
```diff
@@ -164,7 +165,7 @@ export const getRegistrationTasks = () => serverConfigStore.getValue() ? ([
     key: 'dataUserCodeOfConduct',
     completionPropsKey: 'dataUserCodeOfConductCompleted',
     title: 'Data User Code of Conduct',
-    description: <span>Sign the data user code of conduct consenting to the <i>All of Us</i> data use policy.</span>,
+    description: <span>Sign the Data User Code of Conduct consenting to the <i>All of Us</i> data use policy.</span>,
```
Comment (NehaBroad) [inline ui/src/app/pages/homepage/registration-dashboard.tsx:168]: 16

File: ui/src/app/pages/login/account-creation/account-creation-institution.tsx [hunk 8]
```diff
@@ -358,7 +358,7 @@ export class AccountCreationInstitution extends React.Component<Props, State> {
           <div style={styles.institutionalDuaTextBox}>
             For access to the <i>All of Us</i> Research Program data, your institution needs to have signed a Data Use Agreement
             with the program. The institutions listed below have an Institutional Data Use Agreement with the program that
-            enables us to provide their researchers with access to the Workbench.
+            enables us to provide their researchers with access to the Researcher Workbench.
```
Comment (NehaBroad) [inline ui/src/app/pages/login/account-creation/account-creation-institution.tsx:361]: row # 10

File: ui/src/app/pages/login/account-creation/account-creation-options.tsx [hunk 9]
```diff
@@ -78,7 +78,7 @@ export const AccountCreationOptions = {
       value: InstitutionalRole.MIDCAREER},
     {label: `Late career tenured researcher`,
       value: InstitutionalRole.LATECAREER},
-    {label: `Project Personnel (eg: Research Assistant, Data Analyst, Project Manager, Research Coordinator or other roles)`,
+    {label: `Project Personnel (eg: Research Assistant, Data Analyst, Project Manager, Research Coordinator, or other roles)`,
```
Comment (NehaBroad) [inline ui/src/app/pages/login/account-creation/account-creation-options.tsx:81]: Row # 11

File: ui/src/app/pages/login/account-creation/account-creation-options.tsx [hunk 10]
```diff
@@ -138,7 +138,7 @@ export const AccountCreationOptions = {
       roles: Object.keys(InstitutionalRole).map(k => InstitutionalRole[k])}
   ],
   race: [
-    {label: `American Indian or Alaska Native (AIAN)`, value: Race.AIAN},
+    {label: `American Indian or Alaska Native (AI/AN)`, value: Race.AIAN},
```
Comment (NehaBroad) [inline ui/src/app/pages/login/account-creation/account-creation-options.tsx:141]: Row # 12

File: ui/src/app/pages/login/account-creation/account-creation-options.tsx [hunk 11]
```diff
@@ -170,7 +170,7 @@ export const AccountCreationOptions = {
     {label: 'Never attended school/no formal education', value: Education.NOEDUCATION},
     {label: 'Primary/Middle School/High School (Grades 1 through 12/GED)', value: Education.GRADES112},
     {label: 'Some college, Associate Degree or ' +
-      'Technical school (1 to 3 years) or current undergraduate student', value: Education.UNDERGRADUATE},
+      'Technical school (1 to 3 years), or current undergraduate student', value: Education.UNDERGRADUATE},
```
Comment (NehaBroad) [inline ui/src/app/pages/login/account-creation/account-creation-options.tsx:173]: Row # 13

File: ui/src/app/pages/login/account-creation/account-creation-tos.tsx [hunk 12]
```diff
@@ -85,7 +85,7 @@ export class AccountCreationTos extends React.Component<
             </div>
             <div style={{fontWeight: 400}}>
                 By clicking below, or continuing with the registration process or accessing the
-                Workbench, You agree to these Terms and make the following certifications:
+                Researcher Workbench, you agree to these terms and make the following certifications:
```
Comment (NehaBroad) [inline ui/src/app/pages/login/account-creation/account-creation-tos.tsx:88]: Row # 7 and # 8 and # 9

File: ui/src/app/pages/workspace/workspace-edit-text.tsx [hunk 13]
```diff
@@ -11,8 +11,8 @@ import * as React from 'react';
 
 
 export const toolTipTextDemographic = 'For example, by stratifying results based on race/ethnicity, age, ' +
-    'sex, gender identity, sexual orientation, geography disability status, access to care, ' +
-    'education level or income\n';
+    'sex, gender identity, sexual orientation, geography, disability status, access to care, ' +
+    'education level, or income\n';
```
Comment (NehaBroad) [inline ui/src/app/pages/workspace/workspace-edit-text.tsx:15]: Row # 5 and 39

File: ui/src/app/pages/workspace/workspace-edit-text.tsx [hunk 14]
```diff
@@ -40,15 +40,15 @@ export const toolTipTextStigmatization = <div>
 export const ResearchPurposeDescription =
     <div style={{display: 'inline'}}>The <AouTitle/> requires each data user of the <AoU/> data to
       provide a meaningful description of the intended purpose of data use for
-      each workspace they create. To provide transparency to <AoU/> Program participants, your
+      each workspace they create. To provide transparency to <AouTitle/> participants, &nbsp;your
```
Comment (NehaBroad) [inline ui/src/app/pages/workspace/workspace-edit-text.tsx:43]: Row # 22

File: ui/src/app/pages/workspace/workspace-edit-text.tsx [hunk 15]
```diff
@@ -40,15 +40,15 @@ export const toolTipTextStigmatization = <div>
...
       answers below will be made available publicly in the <a target='_blank'
       href='https://www.researchallofus.org/research-projects-directory/'>Research Hub
       Directory </a> on our public website. <b>Your responses will not be used to make decisions about data access.</b>
       <hr/>
-      <i>Note that you are required to create separate Workspaces for each project
-        for which you access <AoU/> data, hence the responses below are expected
+      <i>Note that you are required to create separate workspaces for each project
+        for which you access </i> All of Us <i>data, hence the responses below are expected
```
Comment (NehaBroad) [inline ui/src/app/pages/workspace/workspace-edit-text.tsx:49]: row # 3

File: ui/src/app/pages/workspace/workspace-edit-text.tsx [hunk 16]
```diff
@@ -71,7 +71,7 @@ export const ResearchPurposeItems: Array<ResearchPurposeItem> = [
     longDescription: <div>The primary purpose of the use of <i>All of Us</i> data is to develop
       and/or validate specific methods/tools for analyzing or interpreting data (e.g. statistical
       methods for describing data trends, developing more powerful methods to detect
-      gene-environment or other types of interactions in genome-wide association studies).</div>
+      gene-environment, or other types of interactions in genome-wide association studies).</div>
```
Comment (NehaBroad) [inline ui/src/app/pages/workspace/workspace-edit-text.tsx:74]: Row # 24

File: ui/src/app/pages/workspace/workspace-edit-text.tsx [hunk 17]
```diff
@@ -81,7 +81,7 @@ export const ResearchPurposeItems: Array<ResearchPurposeItem> = [
   }, {
     shortName: 'ancestry',
     shortDescription: 'Genetic Research',
-    longDescription: <div>Research concerning genetics (i.e. the study of genes, genetic variations
+    longDescription: <div>Research concerning genetics (i.e. the study of genes, genetic variations,
```
Comment (NehaBroad) [inline ui/src/app/pages/workspace/workspace-edit-text.tsx:84]: Row # 25

File: ui/src/app/pages/workspace/workspace-edit-text.tsx [hunk 18]
```diff
@@ -92,11 +92,11 @@ export const ResearchPurposeItems: Array<ResearchPurposeItem> = [
     shortName: 'populationHealth',
     shortDescription: 'Population Health/Public Health Research',
     longDescription: <div>The primary purpose of using <i>All of Us</i> data is to investigate
-      health behaviors, outcomes, access and disparities in populations.</div>
+      health behaviors, outcomes, access, and disparities in populations.</div>
```
Comment (NehaBroad) [inline ui/src/app/pages/workspace/workspace-edit-text.tsx:95]: Row # 26

File: ui/src/app/pages/workspace/workspace-edit-text.tsx [hunk 19]
```diff
@@ -92,11 +92,11 @@ export const ResearchPurposeItems: Array<ResearchPurposeItem> = [
...
   }, {
     shortName: 'ethics',
     shortDescription: 'Ethical, Legal, and Social Implications (ELSI) Research',
-    longDescription: <div>This research focuses on ethical, legal and social implications (ELSI)
+    longDescription: <div>This research focuses on ethical, legal, and social implications (ELSI)
```
Comment (NehaBroad) [inline ui/src/app/pages/workspace/workspace-edit-text.tsx:99]: Row # 27

File: ui/src/app/pages/workspace/workspace-edit-text.tsx [hunk 20]
```diff
@@ -126,7 +126,7 @@ export const PrimaryPurposeItems: Array<ResearchPurposeItem> = [ {
   shortName: 'otherPurpose',
   shortDescription: 'Other Purpose',
   uniqueId: 'other-purpose',
-  longDescription: <div>If your Purpose of Use is different from the options listed above, please
+  longDescription: <div>If your purpose of use is different from the options listed above, please
```
Comment (NehaBroad) [inline ui/src/app/pages/workspace/workspace-edit-text.tsx:129]: Row # 28

File: ui/src/app/pages/workspace/workspace-edit-text.tsx [hunk 21]
```diff
@@ -137,9 +137,9 @@ export const toolTipText = {
     You can share this Workspace with other users, allowing them to view or edit your work. Your
     Workspace is where you will go to build concept sets and cohorts and launch Notebooks for
     performing analyses on your cohorts. </div>,
-  cdrSelect: <div>The Curated Data Repository (CDR) is where research data from the <i>All of Us</i>
-    Research Program is stored. The CDR is periodically updated as new data becomes available for
-    use. You can select which version of the CDR you wish to query in this Workspace.</div>,
+  cdrSelect: <div>The Curated Data Repository (CDR) is where research data from the <AouTitle/> is
+    stored. The CDR is periodically updated as new data becomes available for
+    use. You can select which version of the CDR you wish to query in this workspace.</div>,
```
Comment (NehaBroad) [inline ui/src/app/pages/workspace/workspace-edit-text.tsx:142]: Row # 50

File: ui/src/app/pages/workspace/workspace-edit-text.tsx [hunk 22]
```diff
@@ -160,72 +160,72 @@ export const researchPurposeQuestions: Array<ResearchPurposeQuestion> = [
     header: <div> Please provide a summary of your research purpose by responding to the
       questions below.</div>,
     description: <div>Your responses should cover the major components of a research summary:
-      hypothesis, methods and anticipated findings. Therefore, please
+      hypothesis, methods, and anticipated findings. Therefore, please
```
Comment (NehaBroad) [inline ui/src/app/pages/workspace/workspace-edit-text.tsx:163]: Row # 29

File: ui/src/app/pages/workspace/workspace-edit-text.tsx [hunk 23]
```diff
@@ -160,72 +160,72 @@ export const researchPurposeQuestions: Array<ResearchPurposeQuestion> = [
...
       provide sufficiently detailed responses in plain language (without jargon), using as few
       technical terms as possible. </div>
   }, {
     header: 'What are the specific scientific question(s) you intend to study, and why is the ' +
-    'question important (i.e. relevant to science or public health)? \n',
+    'question important (i.e. relevance to science or public health)? \n',
```
Comment (NehaBroad) [inline ui/src/app/pages/workspace/workspace-edit-text.tsx:168]: Row # 30

File: ui/src/app/pages/workspace/workspace-edit-text.tsx [hunk 24]
```diff
@@ -160,72 +160,72 @@ export const researchPurposeQuestions: Array<ResearchPurposeQuestion> = [
...
     description: <div>If you are exploring the data at this stage to formalize a specific
       research question, please describe the reason for exploring the data, and the scientific
       question you hope to be able to answer using the data. <br/>
-      (Free text; 1000 Character limit)</div>
+      (Free text; 1000 character limit)</div>
```
Comment (NehaBroad) [inline ui/src/app/pages/workspace/workspace-edit-text.tsx:172]: Row # 32

File: ui/src/app/pages/workspace/workspace-edit-text.tsx [hunk 25]
```diff
@@ -160,72 +160,72 @@ export const researchPurposeQuestions: Array<ResearchPurposeQuestion> = [
...
   }, {
     header: 'What are the scientific approaches you plan to use for your study? Describe the ' +
-    'datasets, research methods and tools you will use to answer your scientific question(s). \n',
-    description: <div>Free text; 1000 Character limit</div>
+    'datasets, research methods, and tools you will use to answer your scientific question(s). \n',
```
Comment (NehaBroad) [inline ui/src/app/pages/workspace/workspace-edit-text.tsx:175]: Row # 31

File: ui/src/app/pages/workspace/workspace-edit-text.tsx [hunk 26]
```diff
@@ -160,72 +160,72 @@ export const researchPurposeQuestions: Array<ResearchPurposeQuestion> = [
...
+    description: <div>Free text; 1000 character limit</div>
```
Comment (NehaBroad) [inline ui/src/app/pages/workspace/workspace-edit-text.tsx:176]: Row # 32

File: ui/src/app/pages/workspace/workspace-edit-text.tsx [hunk 27]
```diff
@@ -160,72 +160,72 @@ export const researchPurposeQuestions: Array<ResearchPurposeQuestion> = [
...
       to help us assess how well we are serving underrepresented communities—and to protect against
-      potentially stigmatizing research findings, to which these groups are more vulnerable.
-
-      <i>All of Us</i> supports well-designed and responsible research that addresses questions relevant to
+      potentially stigmatizing research findings, to which these groups are more
+      vulnerable. <AoU/> supports well-designed and responsible research that addresses questions relevant to
       underrepresented communities, as well as research that compares different groups and
       populations. At the same time, we recognize that research with underrepresented populations
-      can unintentionally result in harms, including the exacerbation of stigma. We encourage
+      can unintentionally result in harm, including the exacerbation of stigma. We encourage
```
Comment (NehaBroad) [inline ui/src/app/pages/workspace/workspace-edit-text.tsx:204]: Row # 35

File: ui/src/app/pages/workspace/workspace-edit-text.tsx [hunk 28]
```diff
@@ -160,72 +160,72 @@ export const researchPurposeQuestions: Array<ResearchPurposeQuestion> = [
...
       researchers to consider whether their research design, methods, and analyses could
       potentially cause harm to individuals, groups, and communities.</div>
   }, {
     header: 'Request for Review of Research Purpose Description',
-    description: <div>Any research that focuses on certain population characteristics or
+    description: <span>
```
Comment (NehaBroad) [inline ui/src/app/pages/workspace/workspace-edit-text.tsx:209]: This is NOT being used anywhere, i will remove it in another PR. The same text is in workspace-edit.tsx

File: ui/src/app/pages/workspace/workspace-edit-text.tsx [hunk 29]
```diff
@@ -248,8 +248,8 @@ export const SpecificPopulationItems: Array<SpecificPopulationItem> = [
     'African; Middle Eastern or North African (MENA); Native Hawaiian or Other Pacific ' +
     'Islander (NHPI); Hispanic, Latino, or Spanish (H/L/S); Multi-Ancestry (2+ Races)',
     subCategory: [{label: 'Asian', shortName: SpecificPopulationEnum.RACEASIAN},
-      {label: 'Black, African or African American', shortName: SpecificPopulationEnum.RACEAA},
-      {label: 'Hispanic, Latino or Spanish', shortName: SpecificPopulationEnum.RACEHISPANIC},
+      {label: 'Black, African, or African American', shortName: SpecificPopulationEnum.RACEAA},
```
Comment (NehaBroad) [inline ui/src/app/pages/workspace/workspace-edit-text.tsx:251]: Row # 37

File: ui/src/app/pages/workspace/workspace-edit-text.tsx [hunk 30]
```diff
@@ -303,7 +303,7 @@ export const SpecificPopulationItems: Array<SpecificPopulationItem> = [
     ubrDescription: 'Gay; Lesbian; Bisexual; Queer; Other Sexual Orientation Choices',
     subCategory: [
       {
-        label: 'Participants who identify as asexual, bisexual, gay or lesbian or something else ' +
+        label: 'Participants who identify as asexual, bisexual, gay or lesbian, or something else ' +
```
Comment (NehaBroad) [inline ui/src/app/pages/workspace/workspace-edit-text.tsx:306]: Row # 38

File: ui/src/app/pages/workspace/workspace-edit-text.tsx [hunk 31]
```diff
@@ -371,7 +371,7 @@ export const disseminateFindings = [
   },
   {
     label: 'Presentation at community forums or advisory groups (such as town halls, advocacy group ' +
-    'meetings or community advisory boards)',
+    'meetings, or community advisory boards)',
```
Comment (NehaBroad) [inline ui/src/app/pages/workspace/workspace-edit-text.tsx:374]: Row # 33

File: ui/src/app/pages/workspace/workspace-edit.tsx [hunk 32]
```diff
@@ -586,7 +586,7 @@ export const WorkspaceEdit = fp.flow(withRouteConfigData(), withCurrentWorkspace
     renderHeader() {
       switch (this.props.routeConfigData.mode) {
         case WorkspaceEditMode.Create:
-          return 'Create a new Workspace';
+          return 'Create a new workspace';
```
Comment (NehaBroad) [inline ui/src/app/pages/workspace/workspace-edit.tsx:589]: Row # 19

File: ui/src/app/pages/workspace/workspace-edit.tsx [hunk 33]
```diff
@@ -949,7 +949,7 @@ export const WorkspaceEdit = fp.flow(withRouteConfigData(), withCurrentWorkspace
         </WorkspaceEditSection>
         }
         {(!this.isMode(WorkspaceEditMode.Edit) || this.props.workspace.accessLevel === WorkspaceAccessLevel.OWNER) &&
-          <WorkspaceEditSection header={<div><i>All of Us</i> Billing account</div>}
+          <WorkspaceEditSection header={<div><AoU/> billing account</div>}
```
Comment (NehaBroad) [inline ui/src/app/pages/workspace/workspace-edit.tsx:952]: Row # 20

File: ui/src/app/pages/workspace/workspace-edit.tsx [hunk 34]
```diff
@@ -1104,7 +1104,7 @@ export const WorkspaceEdit = fp.flow(withRouteConfigData(), withCurrentWorkspace
               underrepresented populations, either on their own or in comparison to other groups.</label>
           </div>
           <div style={{...styles.text, marginLeft: '1.2rem'}}>
-            <strong>If "Yes": </strong> please indicate your underrepresented population(s) of
+            <strong>If "Yes,"</strong>&nbsp;please indicate your underrepresented population(s) of
```
Comment (NehaBroad) [inline ui/src/app/pages/workspace/workspace-edit.tsx:1107]: Row # 36

File: ui/src/app/pages/workspace/workspace-edit.tsx [hunk 35]
```diff
@@ -1173,7 +1173,7 @@ export const WorkspaceEdit = fp.flow(withRouteConfigData(), withCurrentWorkspace
             </TooltipTrigger>
             &nbsp;If you are concerned that your research could inadvertently stigmatize
             participants or communities, or if you are unsure, let us know. We encourage you to
-            request a review of your research purpose statement by the All of Us Resource Access
+            request a review of your research purpose statement by the <AoU/> Resource Access
```
Comment (NehaBroad) [inline ui/src/app/pages/workspace/workspace-edit.tsx:1176]: Row # 40

File: ui/src/app/pages/workspace/workspace-edit.tsx [hunk 36]
```diff
@@ -1329,7 +1329,7 @@ export const WorkspaceEdit = fp.flow(withRouteConfigData(), withCurrentWorkspace
                 <a style={{color: colors.accent}}
                   href='https://www.researchallofus.org/research-projects-directory/'
                   target='_blank'> displayed publicly </a>
-                 to inform <i>All of Us</i> Research participants. Therefore, please verify that you have provided sufficiently detailed
+                 to inform <AoU/> research participants. Therefore, please verify that you have provided sufficiently detailed
```
Comment (NehaBroad) [inline ui/src/app/pages/workspace/workspace-edit.tsx:1332]: Row # 41

File: ui/src/app/pages/workspace/workspace-library.tsx [hunk 37]
```diff
@@ -88,7 +88,7 @@ const libraryTabs = {
   DEMO_PROJECTS: {
     title: 'Demonstration Projects',
     description: <div>Workspaces included here are meant to demonstrate end-to-end analyses that can be performed
-      using <AoU/> Data. The current set of Research Example Analyses were developed as part of <AoU/> Demonstration
+      using <AoU/> Data. The current set of research example analyses were developed as part of <AoU/> Demonstration
```
Comment (NehaBroad) [inline ui/src/app/pages/workspace/workspace-library.tsx:91]: Row # 49

File: ui/src/assets/json/help-sidebar.json [hunk 38]
```diff
@@ -9,15 +9,15 @@
     {
       "title": "Concept Sets",
       "content": [
-        "Concepts describe information in a patient’s medical record, such as a condition they have, a prescription they are taking or their physical measurements.",
+        "Concepts describe information in a patient’s medical record, such as a condition they have, a prescription they are taking, or their physical measurements.",
```
Comment (NehaBroad) [inline ui/src/assets/json/help-sidebar.json:12]: Row # 42

File: ui/src/assets/json/help-sidebar.json [hunk 39]
```diff
@@ -9,15 +9,15 @@
...
         "Subject areas such as conditions, drugs, measurements, etc. are called \"domains.\"",
         "Concept sets are collections of concepts from a particular domain, that users can save and use to create a dataset for analysis."
       ]
     },
     {
       "title": "Datasets",
       "content": [
-        "Datasets are analysis-ready tables that can be exported to analysis tools such as Notebooks.",
+        "Datasets are analysis-ready tables that can be exported to analysis tools such as notebooks.",
```
Comment (NehaBroad) [inline ui/src/assets/json/help-sidebar.json:20]: Row # 43

File: ui/src/assets/json/help-sidebar.json [hunk 40]
```diff
@@ -30,7 +30,7 @@
     {
       "title": "Data Type",
       "content": [
-        "This table displays details about cohorts, concept sets, and datasets within the workspace, including the name, owner and last changed date."
+        "This table displays details about cohorts, concept sets, and datasets within the workspace, including the name, owner, and last changed date."
```
Comment (NehaBroad) [inline ui/src/assets/json/help-sidebar.json:33]: Row # 44

File: ui/src/app/pages/workspace/workspace-edit-text.tsx [hunk 41]
```diff
@@ -160,72 +160,72 @@ export const researchPurposeQuestions: Array<ResearchPurposeQuestion> = [
...
-      research outcome may fit into the <i>All of Us</i> Research Program Scientific Framework.
+    header: <div>The <AoU/> Research Program would like to understand how (or if)  your
+      research outcome may fit into the <AoU/> Research Program Scientific Framework.
       Please select all of the statements below that describe the outcomes you anticipate from your
       research.</div>,
     description: ''
   }, {
     header: 'Population of interest',
-    description: <div>A primary aim of <i>All of Us</i> is to engage communities that have been
+    description: <div>A primary aim of <AoU/> is to engage communities that have been
```
Comment (NehaBroad) [inline ui/src/app/pages/workspace/workspace-edit-text.tsx:197]: Row # 4

File: ui/src/app/pages/workspace/workspace-edit-text.tsx (hunk 21 above)
Comment (NehaBroad) [reply ui/src/app/pages/workspace/workspace-edit-text.tsx:142]: and Row # 6

File: ui/src/app/pages/workspace/workspace-edit-text.tsx (hunk 24 above)
Comment (jmthibault79) [reply ui/src/app/pages/workspace/workspace-edit-text.tsx:172]: I searched the whole code base for "0 character" to see if there were any other capitalizations like this for e.g. "50 characters"

Nope!  You got them all.

File: ui/src/app/pages/workspace/workspace-edit-text.tsx (hunk 41 above)
Comment (jmthibault79) [reply ui/src/app/pages/workspace/workspace-edit-text.tsx:197]: This looks like the same spacing to me

File: ui/src/app/pages/workspace/workspace-edit-text.tsx [hunk 42]
```diff
@@ -160,72 +160,72 @@ export const researchPurposeQuestions: Array<ResearchPurposeQuestion> = [
...
       historically underrepresented in biomedical research. The next set of questions is designed
       to help us assess how well we are serving underrepresented communities—and to protect against
-      potentially stigmatizing research findings, to which these groups are more vulnerable.
-
-      <i>All of Us</i> supports well-designed and responsible research that addresses questions relevant to
+      potentially stigmatizing research findings, to which these groups are more
+      vulnerable. <AoU/> supports well-designed and responsible research that addresses questions relevant to
```
Comment (NehaBroad) [inline ui/src/app/pages/workspace/workspace-edit-text.tsx:201]: Row # 4

File: ui/src/app/pages/workspace/workspace-edit-text.tsx (hunk 41 above)
Comment (NehaBroad) [reply ui/src/app/pages/workspace/workspace-edit-text.tsx:197]: Wrong change its at line #201

Comment (jmthibault79) [reply ui/src/app/pages/workspace/workspace-edit-text.tsx:197]: Looks like Row #34 is a repeat of this also

File: ui/src/app/pages/workspace/workspace-edit.tsx [hunk 43]
```diff
@@ -443,7 +443,7 @@ export const WorkspaceEdit = fp.flow(withRouteConfigData(), withCurrentWorkspace
 
     renderBillingDescription() {
       return <div>
-        The <i>All of Us</i> Program provides $300 in free credits per user. Please refer to
+        The <AouTitle/> provides $300 in free credits per user. Please refer to
```
Comment (jmthibault79) [inline ui/src/app/pages/workspace/workspace-edit.tsx:446]: Row 21

File: ui/src/app/pages/homepage/recent-workspaces.tsx (hunk 4 above)
Comment (jmthibault79) [reply ui/src/app/pages/homepage/recent-workspaces.tsx:49]: Please update line 51 also

**Pull request merged**